src/views/HLSView.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useRef, Fragment } from "react";
+import {
+  isMobileDevice,
+  selectHLSState,
+  selectPeers,
+  useHMSStore,
+} from "@100mslive/hms-video-react";
+import Hls from "hls.js";
+import { getBlurClass } from "../common/utils";
+import { ChatView } from "./components/chatView";
+
+const defaultClasses = {
+  HLSVideo: "w-full contain",
+};
+
+export const HLSView = ({ isChatOpen, toggleChat, isParticipantListOpen }) => {
+  const videoRef = useRef(null);
+  const peers = useHMSStore(selectPeers);
+  const hlsState = useHMSStore(selectHLSState);
+  useEffect(() => {
+    if (Hls.isSupported()) {
```

Comment:
add null check for videoref.current. Make this if Hls.isSupported && hlsState.url